### PR TITLE
Remove --cadvisor-port - has been deprecated since v1.10

### PR DESCRIPTION
**Pull request by dims**
Related to https://github.com/kubernetes/kubernetes/pull/65707

### CODE REVIEW

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 1]
```diff
@@ -326,7 +326,7 @@ Environment="KUBELET_SYSTEM_PODS_ARGS=--pod-manifest-path=/etc/kubernetes/manife
 Environment="KUBELET_NETWORK_ARGS=--network-plugin=cni --cni-conf-dir=/etc/cni/net.d --cni-bin-dir=/opt/cni/bin"
 Environment="KUBELET_DNS_ARGS=--cluster-dns=10.96.0.10 --cluster-domain=cluster.local"
 Environment="KUBELET_AUTHZ_ARGS=--authorization-mode=Webhook --client-ca-file=/etc/kubernetes/pki/ca.crt"
-Environment="KUBELET_CADVISOR_ARGS=--cadvisor-port=0"
+Environment="KUBELET_CADVISOR_ARGS="
```
Comment (tfogo) [inline content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md]: Can we just remove this entirely?

**Comment by neolit123** [comment]
@kubernetes/sig-cluster-lifecycle-pr-reviews

**Pull request merged**